docs/examples/Track_Stitcher_Example.py

```diff
@@ -13,9 +13,9 @@
 # Track Stitching considers a set of broken fragments of track (which we call tracklets), and aims
 # to identify which fragments should be stitched (joined) together to form one track. This is done
 # by considering the state of a tracked object and predicting its state at a future (or past) time.
-# This example generates a set of `tracklets`, before applying track stitching. The figure below
-# visualizes the aim of track stitching: taking a set of tracklets (left, black) and producing a set
-# of tracks (right, blue/red).
+# This example generates a set of `tracklets` and applies track stitching to them. The figure below
+# visualizes the aim of track stitching: taking a set of tracklets (left, black) and producing a
+# set of tracks (right, blue/red).
 
 # %%
 # .. image:: ../_static/track_stitching_basic_example.png
```

Comment:
Is it possible to make the images a bit larger? They look quite small.

Reply:
Yes but in the interest of the size of this PR, I don't want to do that here. Would rather get the current changes through first, and can make other changes later on :)